Model/Import/Category.php

```diff
@@ -335,7 +332,13 @@ public function getAttributeOptions(AbstractAttribute $attribute): array
 
         if ($attribute->usesSource()) {
             // should attribute has index (option value) instead of a label?
-            $index = in_array($attribute->getAttributeCode(), $this->indexValueAttributes) ? 'value' : 'label';
+            $index = 'label';
+            if (
+                in_array($attribute->getAttributeCode(), $this->indexValueAttributes) ||
+                $attribute->getSourceModel() == "Magento\Eav\Model\Entity\Attribute\Source\Boolean"
```

Comment:
Please use `::class` syntax and also import the class to match the general code style.